src/services/messengerSync/consumer.js

```diff
@@ -8,29 +9,58 @@ const QUEUE_EXTERNAL = Configuration.get('RABBITMQ_MATRIX_QUEUE_EXTERNAL');
 
 let channel;
 
-const validateMessage = (content) => (content.userId && (content.courses || content.teams || content.fullSync));
+const validateMessage = (content) => {
+	if (!content.action) {
+		return false;
+	}
+
+	if (!content.userId && !content.schoolId) {
+		return false;
+	}
+
+	return (content.courses || content.teams || content.fullSync);
```

Comment:
add logging in error cases, if you just return false here. for accepted cases there is no logging required. https://github.com/schul-cloud/schulcloud-server/blob/develop/src/logger/index.js this is probably a `warning`